src/components/definition-list/dt.component.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StyledDt } from './definition-list.style';
+
+const Dt = ({
+  pr = 3,
+  children = 'Label',
```

Comment:
I think the `children` prop should be marked as `isRequired` instead of a default value

Reply:
Agreed and consider it done.

Reply:
✅